src/bokeh/io/webdriver.py

```diff
@@ -84,6 +84,9 @@ def create_chromium_webdriver(extra_options: list[str] | None = None) -> WebDriv
         for op in extra_options:
             options.add_argument(op)
 
+    if os.getenv("BOKEH_IN_DOCKER") == "1":
+        options.add_argument("--no-sandbox")
```

Comment:
This could be covered with a unit test that monkey patches `environ`

Reply:
Just to note, not sure if it would simplify anything, but allegedly this:
```sh
cat /proc/1/cpuset | grep '^/docker'
```
can be used to detect if running in docker or not.

Reply:
We'd have to wrap it in a check that we're not running on Windows so I think that makes it more complicated.